tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/AbstractTychoMapping.java

```diff
@@ -71,16 +71,6 @@ public abstract class AbstractTychoMapping implements Mapping, ModelReader {
     private File multiModuleProjectDirectory;
     private String snapshotFormat;
 
-    @Override
-    public File locatePom(File dir) {
```

Comment:
I don't think we should remove this and instead introduce a useless pom.xml mapping....

Reply:
Was just possible with the higher prioritized PomXMLMapping, now it is reverted.